src/components/Tooltip/BaseTooltip/index.tsx

```diff
@@ -184,6 +185,18 @@ function Tooltip(
         setIsVisible(false);
     }, []);
 
+    const child = useMemo(() => getFirstValue(children), [children]);
```

Comment:
Won't the overhead of the memoization mechanism itself be bigger than the cost of executing this oneliner?

Reply:
Decided to memo it in order to keep potential compatibility with `children` as they are might or may not be memoized - but skipping the `useMemo` will ensure they are never memoized 👍